### PR TITLE
SCMOD-4380: Update Copyright statements to 2018

### PR DESCRIPTION
**Pull request by thilagasanthosh**
https://jira.autonomy.com/browse/SCMOD-4380

Hi @dermot-hardy 
Please review the pull request for story "SCMOD-4380-Update Copyright statements to 2018 before release" and let me know in case of any comments.
Thilaga

### CODE REVIEW

File: job-service-caller/pom.xml [hunk 1]
```diff
@@ -60,7 +60,7 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <autoPull>always</autoPull>
+                    <autoPull>on</autoPull>
```
Comment (dermot-hardy) [inline job-service-caller/pom.xml]: Remove the changes which are unrelated to this Jira, such as this one.  These can be pushed in a separate commit.  We need to keep commits focused.  See https://github.com/salemove/github-review-helper/blob/master/doc/rules.md#focused.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

**Comment by thilagasanthosh** [comment]
Updated the job-service-caller/pom.xml .

File: docs/pom.xml [hunk 2]
```diff
@@ -76,7 +76,7 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <autoPull>always</autoPull>
+                    <autoPull>on</autoPull>
```
Comment (dermot-hardy) [inline docs/pom.xml]: There are still changes here that are not related to this Jira.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

**Review by dermot-hardy — APPROVED** [review]
